src/component-library/Pages/RSE/PageRSEProtocols.tsx

```diff
@@ -9,6 +9,7 @@ import { RSEProtocolViewModel } from "@/lib/infrastructure/data/view-model/rse";
 import { RSEProtocol } from "@/lib/core/entity/rucio";
 import { NormalTable } from "@/component-library/StreamedTables/NormalTable";
 import { TableStyling } from "@/component-library/StreamedTables/types";
+import useReponsiveHook from "@/component-library/Helpers/ResponsiveHook";
```

Comment:
Hey @esilvaju, could you try using tailwind breakpoints instead? I am trying to deprecate the use-responsive hook because it triggers several re-renders when you change the screen size. Using CSS media queries or Tailwind breakpoints works extremely well as an alternative
https://tailwindcss.com/docs/screens

You can use the default breakpoints and refactor the tailwind classes to indicate on which screen size they should be applied. For example, for large screens you might use `xl:px-4`

Reply:
Sure !

Reply:
@esilvaju do you still need some help with this?

Reply:
Thansk @maany ! check it out.